test/runtests.jl

```diff
@@ -1,125 +1,311 @@
 using Impute
+using Tables
 using Test
 using DataFrames
 using RDatasets
 using Statistics
+using StatsBase
+
+import Impute:
+    Drop,
+    DropObs,
+    DropVars,
+    Interpolate,
+    Fill,
+    LOCF,
+    NOCB,
+    Context,
+    WeightedContext,
+    ImputeError
 
 @testset "Impute" begin
     a = Vector{Union{Float64, Missing}}(1.0:1.0:20.0)
     a[[2, 3, 7]] .= missing
     mask = map(!ismissing, a)
+    ctx = Context(; limit=0.2)
 
     @testset "Drop" begin
-        result = impute(a, :drop; limit=0.2)
-        expected = copy(a)
-        deleteat!(expected, [2, 3, 7])
+        @testset "DropObs" begin
+            result = impute(DropObs(; context=ctx), a)
+            expected = copy(a)
+            deleteat!(expected, [2, 3, 7])
 
-        @test result == expected
+            @test result == expected
+            @test result == Impute.dropobs(a; context=ctx)
+
+            a2 = copy(a)
+            Impute.dropobs!(a2; context=ctx)
+            @test a2 == expected
+        end
+        @testset "DropVars" begin
+            @testset "Matrix" begin
```

Comment:
What is the behaviour of `DropVars` on a vector (`MethodError`?) -- we should test that

Reply:
Yes, it'll throw a `MethodError`.

Reply:
Oops, switching to tables means that isn't true anymore. I've added a fallback to a method error for vectors.